### ISSUE TITLE
Cannot reproduce Zero-shot values for RoBERTa

### ISSUE THREAD

**Issue by NamburiSrinath** [alontalmor/oLMpics#4]
Hi,

Thanks for open sourcing the dataset and the analysis.

I am currently working on a research project which involves probing LMs with oLMpics and am trying to replicate the baseline values for RoBERTa.
 
So, here’s what I am doing:
 
1. Preprocess the stem and replace [MASK] with <mask> (for Roberta)
2. Pass the sentence (‘stem’) to Roberta and get the logits for <mask> (the dimension will be batch_size * 30K)
3. At the position of the <mask>, filter out the tokens corresponding to candidates (i.e batch_size * 30K -> batch_size * 2) (Assume the example is from AGE-COMPARISON and has older, younger)
4. Compare these two and predict the output.
 
This is working decently well for BERT (within the range of 5 points), but for RoBERTa, we are unable to replicate the values. While the reported value is around 68% for AGE-COMPARISON, we got only 49%
 
A few quick questions:

1. How are you picking up the choices for RoBERTa?
**AGE-COMPARISON:** Golder, Gyounger, GOlder, GYounger
So, we are confused on which ones to filter while making the decision
2. Do you think the above-described method is valid and in accordance with your paper?

 
I really appreciate any help in this regard 😊
